### PR TITLE
make a README (and fix is_rate_limited to be correct)

### PR DESCRIPTION
**Pull request by paetling**
review @jsingle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+node penalty-box.js
+```
+
+### Getting Started
+Penalty Box relies on a redis instance and can send metrics to a datadog server.
+Penalty Box knows about the following configuration variables (defaults in parentheses):
+```
+debug: if True, enables additional debug information in logs (false)
+log_requests: if True, logs info on each HTTP request to stdout
+port: port on which to run the Lox HTTP API (80)
```
Comment (jsingle) [inline README.md]: lox

File: README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+redis_host: host of the underlying Redis instance (localhost)
+redis_port: port of the underlying Redis instance (6379)
+port: port on which to run the Lox HTTP API (80)
```
Comment (jsingle) [inline README.md]: lox

File: README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    limit: 60
+    remaining: 59
+    is_rate_limted: false
+    reset: <epoch_time_in_ms_till_rate_limit_is_reset>
+}
+```
+
+Some important information on the response body:
+```
+limit - once set will not change
```
Comment (jsingle) [inline README.md]: "cannot be changed"?

Comment (paetling) [inline README.md]: sure

**Comment by jsingle** [comment]
Nice

**Pull request merged**